src/SUMMARY.md

```diff
@@ -1,12 +1,13 @@
 # Summary
 
-[Home](./README.md)
+[Introduction to Ratatui](./README.md)
 
----
+# User Guide
 
-- [Introduction](./introduction.md)
+- [About this book](./introduction.md)
```

Comment:
Maybe just get rid of this page? I don't really see the purpose. It "feels off" when you compare it to the previous and next page.

Reply:
Or move this content to the first `README.md` link.

Reply:
Agreed - I thought about doing this, but went conservative there and left it, but I agree totally with removing it.